### ISSUE TITLE
Unable to scroll down to the deployment report.

### ISSUE THREAD

**Issue by ggouttesol** [jcaillon/3P#167]
### Description of the Issue
After deploying an application, we cannot scroll down to the end of the report : the cursor is moving down but not the text. To see the last lines of the report, we need to click on "export report to html".

### Steps to Reproduce the Issue
1. Deploy an application.
2. Try to scroll down to the end of the deployment report.

### Expected Behavior
Be able to scroll the whole report.

### Actual Behavior
![p_20170504_113218-iloveimg-resized-iloveimg-compressed](https://cloud.githubusercontent.com/assets/23432181/25739589/ced5de8a-3182-11e7-858e-9d983c55e885.gif)

### Debug Information
3P Version : 1.7.4 beta
N++ version : 7.3.2 32 bits

**Comment by jcaillon** [comment]
Yep, known issue, related to the UI framework. I used a lazy method to have scrollable panel with custom scrollbars which uses an internal panel control. The pb is that the latter as a maximum height <_<
will fix asap... :)
